[35mDEBUG   [0m tests.conftest:conftest.py:51 Running fixture setup: test_id
[35mDEBUG   [0m tests.conftest:conftest.py:57 Running test: test_first_node_to_start_publishes with id: 2024-11-21_04-19-18__fdb62e14-96b2-4803-8e6e-a4e05c280c81
[35mDEBUG   [0m src.steps.common:common.py:18 Running fixture setup: common_setup
[35mDEBUG   [0m src.steps.relay:relay.py:28 Running fixture setup: relay_setup
[35mDEBUG   [0m src.steps.relay:relay.py:34 Running fixture setup: setup_main_relay_nodes
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node1_2024-11-21_04-19-18__fdb62e14-96b2-4803-8e6e-a4e05c280c81__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.216.253
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['60270', '60271', '60272', '60273', '60274']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:436 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:165 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:167 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 60270:60270 -p 60271:60271 -p 60272:60272 -p 60273:60273 -p 60274:60274 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=60272 --rest-port=60270 --tcp-port=60271 --discv5-udp-port=60273 --rest-address=0.0.0.0 --nat=extip:172.18.216.253 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=e4bf304b63de7ddeb8ca7addfbe49f3dbad628f0ea8bbde2c684ecc2f8aa06fe --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=60274 --metrics-logging=true --relay=true
[31m[1mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container e74f75a2e069. Exiting log stream.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.216.253 waku f386c4a788bd80321ddf6cca50ff22ba8146f15a99420e043d83c700fcc597b1
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID f386c4a788bd. Setting up logs at ./log/docker/node1_2024-11-21_04-19-18__fdb62e14-96b2-4803-8e6e-a4e05c280c81__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:179 Started container from image wakuorg/nwaku:latest. REST: 60270
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:60270/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:279 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:60270/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.216.253/tcp/60271/p2p/16Uiu2HAmM6icQ4aYkdMqvDKFMpnPbmTxHTXuLrgSgBYmPpAZb9qN","/ip4/172.18.216.253/tcp/60272/ws/p2p/16Uiu2HAmM6icQ4aYkdMqvDKFMpnPbmTxHTXuLrgSgBYmPpAZb9qN"],"enrUri":"enr:-L24QGCJLLgw1kS6ek5o3IsMYbq96Aqx7v7nIcQgdSClGcCbCl1XSV6loAxdjNHaaMlyOmOKH4KsqlQmQ1YtbjgYpr0CgmlkgnY0gmlwhKwS2P2KbXVsdGlhZGRyc5YACASsEtj9ButvAAoErBLY_QbrcN0DgnJzhQADAQAAiXNlY3AyNTZrMaEDfXTMuwWLB2S7uqLY_qcTZnntmvZG8nQglFRpmekjgMWDdGNwgutvg3VkcILrcYV3YWt1MgE"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:284 REST service is ready !!
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node2_2024-11-21_04-19-18__fdb62e14-96b2-4803-8e6e-a4e05c280c81__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.106.203
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['14908', '14909', '14910', '14911', '14912']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:436 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:165 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:167 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 14908:14908 -p 14909:14909 -p 14910:14910 -p 14911:14911 -p 14912:14912 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=14910 --rest-port=14908 --tcp-port=14909 --discv5-udp-port=14911 --rest-address=0.0.0.0 --nat=extip:172.18.106.203 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=cc3b3b8b82ef2ddacbf8dd0cced29460377142c3791a6d5d1698d5ef6d954db4 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=14912 --metrics-logging=true --relay=true --discv5-bootstrap-node=enr:-L24QGCJLLgw1kS6ek5o3IsMYbq96Aqx7v7nIcQgdSClGcCbCl1XSV6loAxdjNHaaMlyOmOKH4KsqlQmQ1YtbjgYpr0CgmlkgnY0gmlwhKwS2P2KbXVsdGlhZGRyc5YACASsEtj9ButvAAoErBLY_QbrcN0DgnJzhQADAQAAiXNlY3AyNTZrMaEDfXTMuwWLB2S7uqLY_qcTZnntmvZG8nQglFRpmekjgMWDdGNwgutvg3VkcILrcYV3YWt1MgE
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.106.203 waku 3ac88a257cb76d25b0cefce34d4d1c79e5e0c78d0dc61e5690fb1eaa94bceb12
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID 3ac88a257cb7. Setting up logs at ./log/docker/node2_2024-11-21_04-19-18__fdb62e14-96b2-4803-8e6e-a4e05c280c81__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:179 Started container from image wakuorg/nwaku:latest. REST: 14908
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:14908/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:279 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:14908/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.106.203/tcp/14909/p2p/16Uiu2HAmJykxJLjfYWR4wHEMp6H4K3cmaMx8o46HPPi1BJtCiQU5","/ip4/172.18.106.203/tcp/14910/ws/p2p/16Uiu2HAmJykxJLjfYWR4wHEMp6H4K3cmaMx8o46HPPi1BJtCiQU5"],"enrUri":"enr:-L24QBR9pg3dLKIcAgegBuJV7on4rbU_lIf8-w5J3DgYoKreZBqVuqGHoi-PdHL32ZYMC5DfjDlbeRXTYuq9U6OJPEsCgmlkgnY0gmlwhKwSasuKbXVsdGlhZGRyc5YACASsEmrLBjo9AAoErBJqywY6Pt0DgnJzhQADAQAAiXNlY3AyNTZrMaEDXfUCdMei2WxTrFvJFKL-5eWTBZMYILW92348TcNuiJaDdGNwgjo9g3VkcII6P4V3YWt1MgE"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:284 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:14908/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.216.253/tcp/60271/p2p/16Uiu2HAmM6icQ4aYkdMqvDKFMpnPbmTxHTXuLrgSgBYmPpAZb9qN"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.steps.relay:relay.py:46 Running fixture setup: setup_optional_relay_nodes
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node3_2024-11-21_04-19-18__fdb62e14-96b2-4803-8e6e-a4e05c280c81__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.12.54
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['37560', '37561', '37562', '37563', '37564']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:436 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:165 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:167 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 37560:37560 -p 37561:37561 -p 37562:37562 -p 37563:37563 -p 37564:37564 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=37562 --rest-port=37560 --tcp-port=37561 --discv5-udp-port=37563 --rest-address=0.0.0.0 --nat=extip:172.18.12.54 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=fac8dd8ce1155cab84a19e5dc9197aaabee3af59dccecbc5ccea9aeea1c116d9 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=37564 --metrics-logging=true --relay=true --discv5-bootstrap-node=enr:-L24QGCJLLgw1kS6ek5o3IsMYbq96Aqx7v7nIcQgdSClGcCbCl1XSV6loAxdjNHaaMlyOmOKH4KsqlQmQ1YtbjgYpr0CgmlkgnY0gmlwhKwS2P2KbXVsdGlhZGRyc5YACASsEtj9ButvAAoErBLY_QbrcN0DgnJzhQADAQAAiXNlY3AyNTZrMaEDfXTMuwWLB2S7uqLY_qcTZnntmvZG8nQglFRpmekjgMWDdGNwgutvg3VkcILrcYV3YWt1MgE
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.12.54 waku d98b6c8acff8e1c530fb55d598344e447f428502dc5f665776ac4346e5ff7a44
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID d98b6c8acff8. Setting up logs at ./log/docker/node3_2024-11-21_04-19-18__fdb62e14-96b2-4803-8e6e-a4e05c280c81__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:179 Started container from image wakuorg/nwaku:latest. REST: 37560
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:37560/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:279 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:37560/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.12.54/tcp/37561/p2p/16Uiu2HAmR7VtP2uez4we8uTNwfywk3ksb2uSatzu8WGGgwKTgmKc","/ip4/172.18.12.54/tcp/37562/ws/p2p/16Uiu2HAmR7VtP2uez4we8uTNwfywk3ksb2uSatzu8WGGgwKTgmKc"],"enrUri":"enr:-L24QPgwbUC7yFtLgc5qoJE0pta_bDVap8PQJVcfEVXMHUMDcia49JainMZSCTvvBfYAi99A7i0-GwnpXwv-OJu5kgsCgmlkgnY0gmlwhKwSDDaKbXVsdGlhZGRyc5YACASsEgw2BpK5AAoErBIMNgaSut0DgnJzhQADAQAAiXNlY3AyNTZrMaEDuRa9nHgm1KjE_ydxmKRRnHkQIebaXuuqTqFXT654J1-DdGNwgpK5g3VkcIKSu4V3YWt1MgE"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:284 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:37560/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.216.253/tcp/60271/p2p/16Uiu2HAmM6icQ4aYkdMqvDKFMpnPbmTxHTXuLrgSgBYmPpAZb9qN"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node4_2024-11-21_04-19-18__fdb62e14-96b2-4803-8e6e-a4e05c280c81__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.192.151
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['15465', '15466', '15467', '15468', '15469']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:436 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:165 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:167 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 15465:15465 -p 15466:15466 -p 15467:15467 -p 15468:15468 -p 15469:15469 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=15467 --rest-port=15465 --tcp-port=15466 --discv5-udp-port=15468 --rest-address=0.0.0.0 --nat=extip:172.18.192.151 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=f657f6abc6ad81b287e13eae4ee8ef36987ee1ffcf67dddab5ce22fc7b8ceefd --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=15469 --metrics-logging=true --relay=true --discv5-bootstrap-node=enr:-L24QGCJLLgw1kS6ek5o3IsMYbq96Aqx7v7nIcQgdSClGcCbCl1XSV6loAxdjNHaaMlyOmOKH4KsqlQmQ1YtbjgYpr0CgmlkgnY0gmlwhKwS2P2KbXVsdGlhZGRyc5YACASsEtj9ButvAAoErBLY_QbrcN0DgnJzhQADAQAAiXNlY3AyNTZrMaEDfXTMuwWLB2S7uqLY_qcTZnntmvZG8nQglFRpmekjgMWDdGNwgutvg3VkcILrcYV3YWt1MgE
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.192.151 waku 000af9b490d3a3f1021a4932aa8ff52dce6b4744f67b4f58769c0ba1b523fe0a
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID 000af9b490d3. Setting up logs at ./log/docker/node4_2024-11-21_04-19-18__fdb62e14-96b2-4803-8e6e-a4e05c280c81__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:179 Started container from image wakuorg/nwaku:latest. REST: 15465
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:15465/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:279 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:15465/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.192.151/tcp/15466/p2p/16Uiu2HAm9R5DXfiwaDQVkJ4D8EsxeQosuFdYC6LunCUPda6337Wk","/ip4/172.18.192.151/tcp/15467/ws/p2p/16Uiu2HAm9R5DXfiwaDQVkJ4D8EsxeQosuFdYC6LunCUPda6337Wk"],"enrUri":"enr:-L24QHc3hFD65U9_YBsEYosoNNpyYijX1DgNZAqgZVXaUknyLkfNvDwapMEt3qWG9hHhWliuosXMpHdwgU3GiEGRqqsCgmlkgnY0gmlwhKwSwJeKbXVsdGlhZGRyc5YACASsEsCXBjxqAAoErBLAlwY8a90DgnJzhQADAQAAiXNlY3AyNTZrMaECz9xPZGPhhzxeVotopZW-lc5JOV1AXg--ALmKPED3QGWDdGNwgjxqg3VkcII8bIV3YWt1MgE"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:284 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:15465/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.216.253/tcp/60271/p2p/16Uiu2HAmM6icQ4aYkdMqvDKFMpnPbmTxHTXuLrgSgBYmPpAZb9qN"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/node5_2024-11-21_04-19-18__fdb62e14-96b2-4803-8e6e-a4e05c280c81__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.155.119
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['6037', '6038', '6039', '6040', '6041']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:436 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:165 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:167 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 6037:6037 -p 6038:6038 -p 6039:6039 -p 6040:6040 -p 6041:6041 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=6039 --rest-port=6037 --tcp-port=6038 --discv5-udp-port=6040 --rest-address=0.0.0.0 --nat=extip:172.18.155.119 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=7a7a3caffbaecdebb3dfbbdd6bdf9a674aabd53ddcd121f9b9e3dd30d34621d6 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=6041 --metrics-logging=true --relay=true --discv5-bootstrap-node=enr:-L24QGCJLLgw1kS6ek5o3IsMYbq96Aqx7v7nIcQgdSClGcCbCl1XSV6loAxdjNHaaMlyOmOKH4KsqlQmQ1YtbjgYpr0CgmlkgnY0gmlwhKwS2P2KbXVsdGlhZGRyc5YACASsEtj9ButvAAoErBLY_QbrcN0DgnJzhQADAQAAiXNlY3AyNTZrMaEDfXTMuwWLB2S7uqLY_qcTZnntmvZG8nQglFRpmekjgMWDdGNwgutvg3VkcILrcYV3YWt1MgE
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.155.119 waku 86792150e8a7125d363f6972c6fbe419e3a2ff536115544f7f065c4594378923
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID 86792150e8a7. Setting up logs at ./log/docker/node5_2024-11-21_04-19-18__fdb62e14-96b2-4803-8e6e-a4e05c280c81__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:179 Started container from image wakuorg/nwaku:latest. REST: 6037
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:6037/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:279 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:6037/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.155.119/tcp/6038/p2p/16Uiu2HAmDYsEVka74ZiC1HggucdyDXiHhqaTrc2hG586oHVbV7eJ","/ip4/172.18.155.119/tcp/6039/ws/p2p/16Uiu2HAmDYsEVka74ZiC1HggucdyDXiHhqaTrc2hG586oHVbV7eJ"],"enrUri":"enr:-L24QBmbcINf50BHA5XBvvwD_czlet6LMgt5SDxZKaXdw7BpBu32aiCJC5p1dPtb3pJHEGp0NbJu-HWtccPbL8_aKJ4CgmlkgnY0gmlwhKwSm3eKbXVsdGlhZGRyc5YACASsEpt3BheWAAoErBKbdwYXl90DgnJzhQADAQAAiXNlY3AyNTZrMaEDDUonmyU13genkMfrFozgxn6YesdZD9eVfViBz1fNV0uDdGNwgheWg3VkcIIXmIV3YWt1MgE"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:284 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:6037/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.216.253/tcp/60271/p2p/16Uiu2HAmM6icQ4aYkdMqvDKFMpnPbmTxHTXuLrgSgBYmPpAZb9qN"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.steps.relay:relay.py:59 Running fixture setup: subscribe_main_relay_nodes
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:60270/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/1"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:14908/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/1"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.steps.relay:relay.py:64 Running fixture setup: subscribe_optional_relay_nodes
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:37560/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/1"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:15465/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/1"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:6037/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/1"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:60270/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d '{"payload": "UmVsYXkgd29ya3MhIQ==", "contentTopic": "/test/1/waku-relay/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.relay:relay.py:123 Checking that peer NODE_1:wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:60270/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"UmVsYXkgd29ya3MhIQ==","contentTopic":"/test/1/waku-relay/proto","version":0,"timestamp":1732162766673689600,"ephemeral":false}]'
[35mDEBUG   [0m src.steps.relay:relay.py:123 Checking that peer NODE_2:wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:14908/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"UmVsYXkgd29ya3MhIQ==","contentTopic":"/test/1/waku-relay/proto","version":0,"timestamp":1732162766673689600,"ephemeral":false}]'
[35mDEBUG   [0m src.steps.relay:relay.py:123 Checking that peer NODE_3:wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:37560/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"UmVsYXkgd29ya3MhIQ==","contentTopic":"/test/1/waku-relay/proto","version":0,"timestamp":1732162766673689600,"ephemeral":false}]'
[35mDEBUG   [0m src.steps.relay:relay.py:123 Checking that peer NODE_4:wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:15465/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"UmVsYXkgd29ya3MhIQ==","contentTopic":"/test/1/waku-relay/proto","version":0,"timestamp":1732162766673689600,"ephemeral":false}]'
[35mDEBUG   [0m src.steps.relay:relay.py:123 Checking that peer NODE_5:wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:6037/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"UmVsYXkgd29ya3MhIQ==","contentTopic":"/test/1/waku-relay/proto","version":0,"timestamp":1732162766673689600,"ephemeral":false}]'
[32mINFO    [0m src.steps.relay:relay.py:71 WARM UP successful!!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:60270/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d '{"payload": "UmVsYXkgd29ya3MhIQ==", "contentTopic": "/test/1/waku-relay/proto", "timestamp": '$(date +%s%N)'}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.relay:relay.py:123 Checking that peer NODE_1:wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:60270/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"UmVsYXkgd29ya3MhIQ==","contentTopic":"/test/1/waku-relay/proto","version":0,"timestamp":1732162766808850176,"ephemeral":false}]'
[35mDEBUG   [0m src.steps.relay:relay.py:123 Checking that peer NODE_2:wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:14908/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"UmVsYXkgd29ya3MhIQ==","contentTopic":"/test/1/waku-relay/proto","version":0,"timestamp":1732162766808850176,"ephemeral":false}]'
[35mDEBUG   [0m src.steps.relay:relay.py:123 Checking that peer NODE_3:wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:37560/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"UmVsYXkgd29ya3MhIQ==","contentTopic":"/test/1/waku-relay/proto","version":0,"timestamp":1732162766808850176,"ephemeral":false}]'
[35mDEBUG   [0m src.steps.relay:relay.py:123 Checking that peer NODE_4:wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:15465/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"UmVsYXkgd29ya3MhIQ==","contentTopic":"/test/1/waku-relay/proto","version":0,"timestamp":1732162766808850176,"ephemeral":false}]'
[35mDEBUG   [0m src.steps.relay:relay.py:123 Checking that peer NODE_5:wakuorg/nwaku:latest can find the published message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:6037/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F1" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"UmVsYXkgd29ya3MhIQ==","contentTopic":"/test/1/waku-relay/proto","version":0,"timestamp":1732162766808850176,"ephemeral":false}]'
[35mDEBUG   [0m tests.conftest:conftest.py:59 Running fixture teardown: test_setup
[35mDEBUG   [0m tests.conftest:conftest.py:83 Running fixture teardown: close_open_nodes
[35mDEBUG   [0m src.node.waku_node:waku_node.py:223 Stopping container with id f386c4a788bd
[35mDEBUG   [0m src.node.waku_node:waku_node.py:230 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:223 Stopping container with id 3ac88a257cb7
[31m[1mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container f386c4a788bd. Exiting log stream.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:230 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:223 Stopping container with id d98b6c8acff8
[31m[1mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container 3ac88a257cb7. Exiting log stream.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:230 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:223 Stopping container with id 000af9b490d3
[31m[1mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container d98b6c8acff8. Exiting log stream.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:230 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:223 Stopping container with id 86792150e8a7
[31m[1mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container 000af9b490d3. Exiting log stream.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:230 Container stopped.
[35mDEBUG   [0m tests.conftest:conftest.py:98 Running fixture teardown: check_waku_log_errors
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.